spec/index.html

```diff
@@ -536,13 +536,13 @@ <h3>Booleans</h3>
     <h3>RDF Blank Nodes</h3>
     <p>
       <a data-cite="RDF12-CONCEPTS#dfn-blank-node">RDF blank nodes</a> in Turtle are expressed
-      as <code>_:</code> followed by a blank node label which is a series of name characters.
-      The characters in the label are built upon
+      as <code>_:</code> followed by a <a data-cite="RDF12-CONCEPTS#dfn-blank-node-identifier">blank node identifier</a> which is a series of name characters.
```

Comment:
```suggestion
      as <code>_:</code> followed by a <a data-cite="RDF12-CONCEPTS#dfn-blank-node-identifier">blank node identifier</a> which is a series of characters.
```
Minor: "name" is a specific technical point (yes, it was in RDF 1.1 doc). Leave it to the next sentence.